### PR TITLE
Get incidents by query fix 

### PR DESCRIPTION
**Pull request by erezh31**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py [hunk 1]
```diff
@@ -18,23 +17,29 @@ def parse_datetime(datetime_str):
...
+                elif unit == 'months':
+                    number *= 43800
+                    unit = 'minutes'
+
+                kargs = {}
+                kargs[unit] = int(number)
+                result = datetime.now() - timedelta(**kargs)
+                return result
+        except Exception:
+            return None
```
Comment (lperryp) [inline Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py:42]: Perhaps it's better to return an error here? might be confusing

File: Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py [hunk 2]
```diff
@@ -104,19 +109,41 @@ def get_incidents_by_page(args, page, fields_to_populate, include_context):
...
         except Exception:
             pass
-        if from_datetime is None and from_date.strip().endswith("ago"):
-            from_datetime = parse_relative_time(from_date)
+        # try to parse relative time
+        if date_object is None and date_string.strip().endswith("ago"):
+            date_object = parse_relative_time(date_string)
+
+        if date_object:
+            return date_object.astimezone().isoformat('T')
```
Comment (lperryp) [inline Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py:125]: perhaps we can add a UT to verify that this is the sting format supported in XSOAR

**Review by lperryp — APPROVED** [review]

File: Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py (hunk 1 above)
Comment (erezh31) [reply Packs/Base/Scripts/GetIncidentsByQuery/GetIncidentsByQuery.py:42]: I handle this as log error... I don't want to return error - not breaking things

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by kirbles19** [comment]
@erezh31 Done

**Pull request merged**